plugins/inputs/dpdk/README.md

```diff
@@ -57,6 +57,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 ```toml @sample.conf
 # Reads metrics from DPDK applications using v2 telemetry interface.
+# This plugin ONLY supports Linux
```

Comment:
Hmmm according to https://www.dpdk.org/ecosystem/#distributions at least also FreeBSD is supported... I wonder why we are not enabling the plugin for this platform...

Reply:
@zak-pawel when you wrote the DPDK plugin did you only target Linux and not include BSD for any particular reason?

Reply:
@srebhan @powersj 
It was tested on linux only. However, it might also run on freebsd (without any guarantee from our side :))

Reply:
So let's keep this Linux only. I was just wondering as the website explicitly mentions FreeBSD support.